docs/business-apps/power-automate/sharepoint-connector-actions-triggers.md

```diff
@@ -4,237 +4,205 @@ ms.date: 03/13/2020
+Gets the properties saved in the columns in the library for all folders and files stored in the library. You can also filter down to the items that match a condition. To work with the output from this action, use the **Apply to each** section. When using this with the On-Premises Data Gateway, you may need to be manually enter the name of the library to connect to. For more information about this action, visit the following documentation: [In-depth analysis into **Get items** and **Get files** SharePoint actions for flows in Power Automate](../power-automate/guidance/working-with-get-items-and-get-files.md).
 
 ### Get folder metadata
-
 Gets info about the folder. Uses a file identifier to select the folder.
 
 ### Get folder metadata using path
-
 Gets info about the folder. Uses a folder path to select the folder.
 
 ### Get item
-
 Gets a single item by its ID from a SharePoint list.
 
 ### Get items
-
-Gets items from a SharePoint list.
+Gets items from a SharePoint list. For more information about this action, visit the following documentation: [In-depth analysis into **Get items** and **Get files** SharePoint actions for flows in Power Automate](../power-automate/guidance/working-with-get-items-and-get-files.md).
 
 ### Get list views
-
 Gets views from a SharePoint list.
 
 ### Get lists
-
 Gets SharePoint lists from a site.
 
 ### Grant access to an item or a folder
-
 Grants access to an item or a folder in SharePoint to specific people.
 
 ### Join hub site
-
 Joins the requested site to the hub site. An Approval Token is required to complete the join successfully, if that hub requires approval. If applicable, specify the same Approval Correlation Id as used in the **Set hub site join status to pending** action.
 
 ### List folder
-
 Returns files contained in a SharePoint folder.
 
 ### List root folder
-
 Returns files in the root SharePoint folder.
 
 ### Move file
-
 Moves a file. Works similarly to the **Move to** command in SharePoint libraries. Returns info about the new file after its move.
 
 ### Move folder
-
 Moves a folder. Works similarly to the **Move to** command in SharePoint libraries. Returns info about the new folder after its move.
 
 ### Resolve person
-
 Returns a single matching user value so it can be assigned to a column of type person. If there are no matches, or multiple matches, this action errors out.
 
 ### Send an HTTP request to SharePoint
+Constructs a SharePoint REST API to invoke.For more information about this action, visit the following documentation: [Working with the SharePoint Send HTTP Request flow action in Power Automate](../power-automate/guidance/working-with-send-sp-http-request.md).
```

Comment:
Constructs a SharePoint REST API to invoke. For more info about this action, see (don't forget space in between sentences).